.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,24 @@
+---
+name: Run tests
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Clone code
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
+
+      - name: Install build dependencies
+        run: |
+          python -m pip install -r requirements.txt
+
+      - name: Fetch latest packages data and build the website
+          make generate
```

Comment:
https://github.com/meshy/pythonwheels/actions/runs/2447771195

This needed a `run`:

```yml
      - name: Fetch latest packages data and build the website
        run: |
          make generate
```

Reply:
:facepalm: thanks!